Source/SwiftLintFramework/Rules/Idiomatic/UnavailableConditionRule.swift

```diff
@@ -9,7 +9,7 @@ public struct UnavailableConditionRule: ConfigurationProviderRule, AutomaticTest
     public static let description = RuleDescription(
         identifier: "unavailable_condition",
         name: "Unavailable Condition",
-        description: "Use #unavailable instead of #available with an empty body.",
+        description: "Use #unavailable/#available instead of #available/#unavailable with an empty body.",
```

Comment:
The idea of this rule was to catch conditions that could be replaced with `#unavailable`.

Since now it's apparently catching empty `#unavailable` with an `#else`, we should use specific warning messages on the violations for clarity.